### PR TITLE
Feat/#80

### LINKED ISSUE #80: Add Progress View On The Map.
### Explanation:
When the view is changed, it is not possible to know what is happening before getting the location on the map, so it is necessary to place a view with transparency that allows blocking the user's actions on the map before showing the location. it should be displayed for a few seconds until the location is obtained and disappears, it is activated by inserting the ticket number, the necessary resources are available in the Figma file.

### Screenshots
View_map--confirmation_progress

### Checklist

- [ ] place transparency.
- [ ] Place the progress with circular animation, this should be centered so you can see that it is loading the data.
- [ ] Verify that the view can be viewed from desktop and mobile.

### References

(https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=800%3A740)

### PR DESCRIPTION
**Pull request by Eghinner**
### Description

Created and implemented Spinner component in load conditions. Also, light changes to the modal displayed in case of error.

fixes #80 

#### Screenshots

spinner
spinner-responsive

#### Type of change

- [x] Added Spinner component
- [x] Delete unnecessary loading state
- [x] In file replaced Loadingview for Spinner

### How to test it

1. Go to 'delivery/902b7f07-da34-4ab5-9a8c-d9db03ba55b5' or 'delivery/90' to show modal changes

### Checklist:

- [x] place transparency.
- [x] Place the progress with circular animation, this should be centered so you can see that it is loading the data.
- [x] Verify that the view can be viewed from desktop and mobile.

### CODE REVIEW

File: src/components/Modal/Modal.styled.jsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import styled from '@emotion/styled';
 
 export const Overlay = styled.div`
+  z-index: 99999999;
```
Comment (Eghinner) [inline src/components/Modal/Modal.styled.jsx:4]: This is for blocking the user's actions
localhost_3000_delivery_902b7f07-da34-4ab5-9a8c-d9db03ba55b(BlackBerry Z30) (1)
localhost_3000_delivery_902b7f07-da34-4ab5-9a8c-d9db03ba55b(BlackBerry Z30)
.

**Comment by Israel-Laguan** [comment]
Please review with latest dev, linter issues fixed

**Comment by Eghinner** [comment]
> Please review with latest dev, linter issues fixed

done

File: src/pages/Delivery/index.jsx [hunk 2]
```diff
@@ -80,6 +80,7 @@ const Delivery = () => {
     >
       {center[0] && <Map center={center} dasher={dasher} permission={permission} />}
     </Layout>
+    </Fragment>
```
Comment (heliomar-pena) [inline src/pages/Delivery/index.jsx]: Check identation. Please try to run linter

**Review by heliomar-pena — APPROVED** [review]
Good work! just one comment

File: src/pages/Delivery/index.jsx (hunk 2 above)
Comment (Eghinner) [reply src/pages/Delivery/index.jsx]: done

File: src/components/Modal/Modal.styled.jsx (hunk 1 above)
Comment (Israel-Laguan) [reply src/components/Modal/Modal.styled.jsx:4]: A little too much, but effective hehe

**Review by Israel-Laguan — APPROVED** [review]

File: src/pages/Delivery/index.jsx [hunk 3]
```diff
@@ -36,50 +37,50 @@ const Delivery = () => {
...
-    <Layout
-      packageId={id}
-      headerTitle={headerStatus[currentStatus]?.headerTitle}
-      headerSubtitle={headerStatus[currentStatus]?.headerSubtitle}
-      clientAddress={packageInformation?.packages[0]?.client_address}
-      estimatedArrival={packageInformation?.packages[0]?.estimated_arrival}
-      isLoading={isLoading}
-      toChat={toChat}
-      DeliveryConfirmedModal={
+    <Fragment>
```
Comment (e-azocar) [inline src/pages/Delivery/index.jsx]: Why you use `<Fragment>` instead just `<>`?

**Review by e-azocar — APPROVED** [review]

File: src/pages/Delivery/index.jsx (hunk 3 above)
Comment (Eghinner) [reply src/pages/Delivery/index.jsx]: Yeah, no problem. I like that

**Review by heliomar-pena — APPROVED** [review]

**Pull request merged**